src/components/cc-button/cc-button.js

```diff
@@ -115,6 +119,40 @@ export class CcButton extends LitElement {
     this._cancelMode = false;
   }
 
+  /**
+   * The `aria-label` attribute should only be set if the `accessibleName` prop is set or if the button only shows an image with no text.
+   *
+   * @returns {string|undefined} the value of the `aria-label` attribute or `undefined` if the attribute should not be set.
+   */
+  _getAriaLabel () {
+    if (this.accessibleName != null) {
+      return this.accessibleName?.trim() ?? '';
```

Comment:
If accessibleName is not null, should we really use `?.trim()` instead of `.trim()`?